### ISSUE TITLE
fn routes create generates broken reference to existing image 

### ISSUE THREAD

**Issue by shaunsmith** [fnproject/cli#74]
When issuing:
`fn routes create myapp /hello -i shaunsmith/hello-node:0.0.8`
The result is:
```
/hello created with shaunsmith/:0.0.1
```

The image name is missing and the tag is incorrect.

`fn routes list myapp` confirms the route is setup wrong:

```
path		image				endpoint
/hello		shaunsmith/:0.0.1		localhost:8080/r/myapp/hello
```

**Comment by zootalures** [comment]
Took a look at this - fn routes create is still loading the func file and --image doesn't seem to do anything

adding 
name: to the func file allows this to work

**Comment by zootalures** [comment]
closed by #75
